lib/resources/dstu2/patient.yaml

```diff
@@ -27,15 +29,282 @@ fields:
   cardinality: 0..*
   type: BackboneElement
   description: A contact party (e.g. guardian, partner, friend) for the patient.
+  action: terminology
   children:
 
   - name: relationship
     required: 'No'
     cardinality: 0..*
     type: CodeableConcept
     description: The nature of the relationship between the patient and the contact person.
+    action: terminology
     binding:
       description: The nature of the relationship between a patient and a contact person for that patient.
       terminology:
       - display: PatientContactRelationship
         system: http://hl7.org/fhir/patient-contact-relationship
+
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType": "Patient"
+    }
+  note: resourceType must be Patient
+  url: http://hl7.org/fhir/DSTU2/json.html#resources
+
+- name: identifier
+  required: 'Yes'
+  cardinality: 1..*
+  type: Identifier
+  description: The identifiers for the patient.
+  action: create
+  example: |
+    {
+      "identifier": [
+        {
+          "assigner": {
```

Comment:
I thought we also required the identifier type to come in?

Reply:
We do on everything but the first. The first can only contain the assigner.